site/networking.xml

```diff
@@ -105,7 +105,7 @@ Or using classic config format:
 
 <pre class="code">
 listeners.tcp.1 = 127.0.0.1:5672
-listeners.tcp.2 = ::1:5672
+listeners.tcp.2 = [::1]:5672
```

Comment:
I reverted that change due to https://github.com/basho/cuttlefish/issues/216. We'll work on that later.